src/main/scala/components/helpers.scala

```diff
@@ -4,7 +4,223 @@ package dinocpu
 
 import chisel3._
 import chisel3.util._
+import scala.math._
+import scala.collection.mutable.ArrayBuffer
 
+object Util
```

Comment:
Is this all needed? If so, I would make a new file called "util" instead of putting it here. 

I would rather not clutter the DINO CPU if we can avoid it.

Reply:
Will do. After some rewriting I'll see if I can get rid of that file

Reply:
Another bump. At a minimum remove what's not needed. I think `WideCounter` is the only thing I see in your code.

Reply:
There were a lot of implicit functions I was using to avoid explicitly type casting everything, but I went ahead and removed all Util related code and changed csr to have WideCounter and just manually cast things as needed